### ISSUE TITLE
Change The Silly Created Names In Docker Desktop With DateTimeMinute Combination Instead

### ISSUE THREAD

**Issue by Speaktive** [docker/roadmap#214]
**Tell us about your request**
When a container is created, the Docker system creates a unique name for it that shows in the Docker Desktop. The names given are silly combinations that are made to be unique. However, the names are not helpful in ANY way!If you are running multiple containers, the system can quickly become unmanageable because you can't instantly get information about the container in the interface without clicking on it. You just see the silly, created names and have to click through to figure out which is which or remember which port you put each one on. This is cumbersome.

**Which service(s) is this request for?**
Docker Desktop

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
Instead of showing the silly name, can you give users who don't want to see those made-up names an option to have the container name be a combination of the date, hour, minute, seconds & port? This would allow users who have to run multiple containers to easily and quickly be able to tell which containers are which in the GUI without clicking on it. That information is far more useful than the silly name. 
So instead of having a container arbitrarily named **dapper_baking**, the name would be something like **04272021_112234_80**. 

**Are you currently working around the issue?**
Right now, I have to click through each container so I can see which ones are which. I can't tell which ones were created an hour ago versus which ones were created 12 hours ago. That is so unfortunate because I love Docker and those silly names just aren't useful in any way.

**Additional context**
FYI - I LOVE DOCKER!

**Comment by thaJeztah** [comment]
(Thinking out loud)

I think this could be addressed in Docker Dashboard (showing more details in the overview), and if Docker Desktop used a naming-scheme when creating containers using the dashboard. The random-names are generated by the docker daemon if no name is provided, but it's possible to set custom names.

For reference, the `docker` cli allows you to view more details (date created, how long container is running, which image was used).

```bash
docker ps --format 'table {{.ID}}\t{{.CreatedAt}}\t{{.Status}}'
CONTAINER ID   CREATED AT                       STATUS
6abff90a7ffd   2021-04-26 17:55:14 +0200 CEST   Up 2 days
2b6755d8ef7a   2021-04-26 08:55:42 +0200 CEST   Up 3 days
1cd13c4f74d5   2021-04-22 16:02:01 +0200 CEST   Up 6 days
```

While the dashboard shows the "image", it's not a separate column, making it somewhat harder to "spot"


Perhaps Docker Desktop could

- show date created/running since in a separate column
- generate names based on information about the container (if no custom name is provided), e.g. `<image>-<imagetag>-<number>`, which would show something like `ubuntu-latest-01` or `nginx-1.20.0-alpine-01`

**Comment by michal-mazurkiewicz** [comment]
Yeah I agree.. it could at least contains image name included so you know what to run .. .
